src/modules/analysis/AnalysisViewComponent.js

```diff
@@ -58,6 +54,18 @@ const getTableRow = a => {
   );
 };
 
+const showLabelAndValue = ({ id, label, md, value }) => (
```

Comment:
just a tiny change reqeust. Change it to LabelAndValue. Because this is a component and not a function.